frontend/src/app/auth.service.ts

```diff
@@ -47,6 +47,28 @@ export class AuthService {
         }
       });
   }
+  resetPassword(username: {username: string}): Observable<string> {
```

Comment:
Same situation with parameter as commented for method 'confirmResetPassword'

Reply:
Done.